src/defs/policyResultRepresentation.ts

```diff
@@ -0,0 +1,9 @@
+import PolicyRepresentation, {DecisionEffect} from './policyRepresentation';
+import ScopeRepresentation from './scopeRepresentation';
+
+export default interface PolicyResultRepresentation {
+  policy: PolicyRepresentation;
+  status: DecisionEffect;
+  associatedPolicies: PolicyResultRepresentation[];
+  scopes: ScopeRepresentation[];
```

Comment:
@jenny-s51 Do you have a link to the documentation for this type? I cannot seem to find it in the documentation.

I think we also have to mark these fields as optional, as the API does not guarantee they are actually set (which is a nice think we inherit from the design flaws of Java).

Reply:
The type doesn't seem to exist in the docs and can be found here @jonkoops : https://github.com/keycloak/keycloak/blob/bfce612641a70e106b20b136431f0e4046b5c37f/core/src/main/java/org/keycloak/representations/idm/authorization/PolicyEvaluationResponse.java#L119-L124

Reply:
Great, thanks for linking!